### DISCUSSION TITLE
[Help] how can i Hide verification text

### DISCUSSION THREAD

**Discussion by flyingcrp** [ant-design/ant-design#28261] [Q&A] [answered]
i want to hide the form verification error message,just keep the Verified status ,  what should i do ?

- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/jibenshiyong-antd492-forked-j9euo?file=/index.js)

### Steps to reproduce
.

### What is expected?
.

### What is actually happening?
.

| Environment | Info |
|---|---|
| antd | undefined |
| React | 16.13.1 |
| System | windows |
| Browser | chrome  latest |

**Comment by xrkffgg** [comment, marked as answer]
```css
.ant-form-item-explain { 
    display: none;
}
```

**Comment by xrkffgg** [comment]
@afc163 
现在 discussions 开放了。是不是可以引导用户将问题转移到 discussions 里。

这里面可以回答问题，选择答案，也方便以后查阅。

issue 可以直接转移到 discussions 里

**Comment by afc163** [comment]
@xrkffgg 应该的，how to 类问题可以引导到 discussions 里。

**Reply by xrkffgg** [reply]
不知道这块的 API 有了吗，比如把 issue 转成 discussions。
先手动转吧，我看看 API。后续 Usage 的 label 也可以改 成 转移。
ant-bot 加几个关键字

**Reply by afc163** [reply]
提到 Stack Overflow 的地方也都可以加上 discusstions，包括 https://github.com/ant-design/ant-bot/blob/master/app/actions/replyUsage.js#L15 和官方文档里的地方。

**Reply by xrkffgg** [reply]
我先把 链接 加上。

需要自动给转移吗？

**Reply by afc163** [reply]
可以有

**Reply by xrkffgg** [reply]
还没有 相关于 discussions 的 API，等以后 这个功能成熟后再加上。
